README.md

```diff
@@ -189,12 +196,20 @@ def test_model(lsm_project: lsm_project.LsmProject) -> None:
         service_identity_attribute_value=None,
     )
 
-    # Add the service to the mocked server.  From now on it will be taken into account
-    # for EACH compile
+    # Add a service to our inventory
     lsm_project.add_service(service)
```

Comment:
I am wondering if validation and add default shouldn't be options (or even defaults) in add_service?

Reply:
I understand you want to offer the flexibility but in "real" lsm this is also enforced by the api server and not the validation compile

Reply:
I wouldn't make it a default, as it would break our test case where we try to reproduce a specific failure for a given "existing" service (which is not created from scratch.  I can add an option to allow it at least

Reply:
Done in https://github.com/inmanta/pytest-inmanta-lsm/pull/391/commits/7a0d37b1c58d695056bbfe0be5d303ec04d4912a